src/scripts/install_machine.py

```diff
@@ -129,16 +129,16 @@ def prepare_cloud_init(args):
             mode='w+'
         )
         user_data_file.write("#cloud-config\n")
-        if 'userLogin' in args:
+        if args['userLogin']:
```

Comment:
This is already merged in https://github.com/cockpit-project/cockpit-machines/commit/b3681c910006332e1cea2805cbbfc0b7986ae50f

Yet this commit also changes references. Github does not seem to see that as conflict. This is weird. I would say it needs rebase (and the changed refs should be in different commit?)